core/os/os.cpp

```diff
@@ -222,6 +222,23 @@ uint64_t OS::get_embedded_pck_offset() const {
 	return 0;
 }
 
+// Default boot screen rect scale mode is "Keep Aspect Centered"
+Rect2 OS::calculate_boot_screen_rect(const Size2 window_size, const Size2 imgrect_size) const {
+	Rect2 screenrect;
+	if (window_size.width > window_size.height) {
+		//scale horizontally
```

Comment:
```suggestion
		// Scale horizontally.
```

Same for the rest of comments.